incubator/hnc/test/e2e/demo_test.go

```diff
@@ -106,9 +108,9 @@ var _ = Describe("Demo", func() {
 		// at least 20 seconds is needed here from experiments 
 		RunShouldContain("Welcome to nginx!", 20, 
 			"kubectl run client -n", nsService1, clientArgs, cmdln)
-		RunShouldContain("Welcome to nginx!", 10, 
+		RunShouldContain("Welcome to nginx!", tenSecTimeout, 
 			"kubectl run client -n", nsTeamA, clientArgs, cmdln)
-		RunShouldContain("Welcome to nginx!", 10, 
+		RunShouldContain("Welcome to nginx!", tenSecTimeout, 
```

Comment:
BTW here after the first 20s, maybe the other two can be just `defTimeout` instead of 10s?

Reply:
With `defTimeout` this test fails half of the time actually